### PR TITLE
Follow up migirate from icsp to idms

### PR DESCRIPTION
**Pull request by QiWang19**
Address the migration of new crd by clearfying new controller and webhook.
Specify that the new CRDs will not be exposed until the migration works done.

Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

File: enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md [hunk 1]
```diff
@@ -375,16 +387,11 @@ the primary registry of the mirrors.
 
 #### Upgrade Strategy
 
-An existing cluster is required to make an upgrade to 4.11 in order to make use of the allow mirror by tags feature. Before the deprecation of ImageContentSourcePolicy, MCO needs to watch for both ImageContentSourcePolicy, and ImageDigestMirrorSet and create objects.
-
-Tier 1 component `imagecontentsourcepolicy.operator.openshift.io/v1alpha1` will stay 12 months or 3 releases from the announcement of deprecation.
-During the development on the release that one release ahead of deprecating `imagecontentsourcepolicy.operator.openshift.io/v1alpha1`,
-MCO will copy existing ImageContentSourcePolicy objects to ImageDigestMirrorSet and create new objects, and delete the
-ImageContentSourcePolicy objects. If any errors appear during the process, MCO should report `Upgradeable=False`.
+An existing cluster is required to make an upgrade to 4.11 in order to make use of the allow mirror by tags feature.
```
Comment (rphillips) [inline enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md:390]: 4.12, right?

File: enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md [hunk 2]
```diff
@@ -401,17 +408,9 @@ Upgrade skew will not impact this feature. The MCO does not require skew check.
...
-  - [Cluster-config-operator](https://github.com/openshift/cluster-config-operator)
-  - [Openshift-api-server](https://github.com/openshift/openshift-apiserver/blob/98786f917ffc7d3dc3b05893f405970b87a419b9/pkg/image/apiserver/registries/registries.go)
-  - [Runtime utils](https://github.com/openshift/runtime-utils/blob/8b8348d80d1d1e7b6cf06fb009d5965e0b55baa2/pkg/registries/registries.go#L50)
-  - [Openshift-controller-manager](https://github.com/openshift/openshift-controller-manager/blob/2a11f145ad7fcf3e92460800de1d13ba7fbb90b0/pkg/build/controller/build/build_controller.go#L20943)
-
-The Node team will be making changes to the above impacted components to enable the new CRDs. Planning target CY21Q4
+
+This [document](https://docs.google.com/document/d/11FJPpIYAQLj5EcYiJtbi_bNkAcJa2hCLV63WvoDsrcQ/edit?usp=sharing) keeps a list of components that use operator.openshift.io/v1alpha1 ImageContentSourcePolicy. Components must be upgraded to use ImageDigestMirrorSet before the exposure of ImageDigestMirrorSet, ImageTagMirrorSet CRDs.
```
Comment (mrunalp) [inline enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md:412]: Can we list the components here in the enhancement?

**Comment by mrunalp** [comment]
This looks fine to me based on the discussions we have had. I will let @soltysh approve it for merge.

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**